src/OpenTelemetry.Exporter.Prometheus.AspNetCore/PrometheusExporterMiddleware.cs

```diff
@@ -102,4 +110,28 @@ public async Task InvokeAsync(HttpContext httpContext)
 
         this.exporter.OnExport = null;
     }
+
+    private bool AcceptsOpenMetrics(HttpRequest request)
+    {
+        var requestAccept = request.Headers[HeaderNames.Accept];
+
+        if (StringValues.IsNullOrEmpty(requestAccept))
+        {
+            return false;
+        }
+
+        var acceptTypes = requestAccept.ToString().Split(',');
+
+        foreach (var acceptType in acceptTypes)
+        {
+            var acceptSubType = acceptType.Split(';').FirstOrDefault()?.Trim();
```

Comment:
How much memory allocation would this introduce? Could this be improved with a zero-alloc one? (e.g. string matching instead of splitting).

Reply:
Yep, this was copy/pasted from the Java version. I'll optimize this using `MediaTypeHeaderValue`.

Reply:
Updated with zero allocations (hopefully)